UI/MainScreen.cpp

```diff
@@ -1167,11 +1167,11 @@ void MainScreen::CreateViews() {
 		}
 	}
 
-#if !PPSSPP_PLATFORM(UWP)
-	// Having an exit button is against UWP guidelines.
+//#if !PPSSPP_PLATFORM(UWP)
+	// Having an exit button is against UWP guidelines?.
```

Comment:
No, but:
1- Emulators are not allowed anyway in the store (I see the exist button is important to have)
2- Instead of `CoreApplication::Exit()` we are using [TryConsolidateAsync](https://learn.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.applicationview.tryconsolidateasync?view=winrt-22621)

> This method is a programmatic equivalent to a user initiating a close gesture for the app view.

I remember that there was a remarks regarding `CoreApplication::Exit()` but not able to find that anymore.

Reply:
Well, seems reasonable to me then.

-[Unknown]